### PR TITLE
use table format, add automate section

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

File: documentation/vespa-plugins.html [hunk 1]
```diff
@@ -20,55 +20,94 @@
...
+<tr id="automate">
+  <th>Automate</th>
+  <td>
+    <p>
+    Add testing to automate deployments:
+    <ul>
+      <li><em>System Tests</em> for functional tests -
+        a system test is an integration test that tests the application API endpoints</li>
+      <li><em>Staging Tests</em> to validate the instance before and after the change,
+          i.e. two test runs with potentially different tests</li>
```
Comment (jonmv) [inline documentation/vespa-plugins.html:93]: Perhaps something to further distinguish this from the above — staging tests validate that state in the cluster is not broken by an upgrade of the Vespa runtime, or application code or config?

File: documentation/vespa-plugins.html [hunk 2]
```diff
@@ -20,55 +20,94 @@
...
+      <li><em>Production Tests</em> to validate a production instance before upgrading the next instance</li>
+    </ul>
+    System and Staging tests require a (downscaled) full instance,
+    whereas Production tests should be read-only on existing production instances.
```
Comment (jonmv) [inline documentation/vespa-plugins.html:97]: Well, we don't support read access to production clusters in tests in Vespa Cloud (yet?).

Comment (kkraune) [reply documentation/vespa-plugins.html:97]: right, will fix

File: documentation/vespa-plugins.html (hunk 1 above)
Comment (kkraune) [reply documentation/vespa-plugins.html:93]: maybe. It is not _that_ interesting for the user that we also test during vespa upgrades, in the context of this document. can add some more details on code/config

**Comment by kkraune** [comment]
@jonmv ptal

**Review by jonmv — APPROVED** [review]

**Pull request merged**